packages/patterns-react/src/patterns/FeaturedLink/FeaturedLink.js

```diff
@@ -0,0 +1,100 @@
+/**
+ * Copyright IBM Corp. 2016, 2018
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { DDS_FEATURED_LINK } from '../../internal/FeatureFlags';
+import FeaturedLinkItem from './FeaturedLinkItem';
+import { Image } from '@carbon/ibmdotcom-react';
+import PropTypes from 'prop-types';
+import React from 'react';
+import { settings as ddsSettings } from '@carbon/ibmdotcom-utilities';
+import { featureFlag } from '@carbon/ibmdotcom-utilities';
+import { settings } from 'carbon-components';
+
+const { stablePrefix } = ddsSettings;
+const { prefix } = settings;
+/**
+ * Featured Link Component
+ *
+ * @param {object} images props object
+ * @returns {*} FeaturedLink JSX component
+ */
+const sortImages = images => {
```

Comment:
@photodow Are we setting the media values here in the pattern itself or will be let adopters pass it through the `images` prop?

Reply:
If I understand what you're asking they should be passing it through the props. I don't think sorting should be happening in this pattern. We should just be able to simply pass in the images into the `Image` component, right?

```
<Image {...image} />
```

Reply:
right!